src/app/frame/page.tsx

```diff
@@ -1,10 +1,130 @@
+      <div className="flex w-full flex-row justify-between px-4">
+        <PreviousPage target="/" />
+        <Link href="/list" className="pr-4">
+          <SVGGoToList />
+        </Link>
+      </div>
+      <h1 className="text-xl">AI 프로필이 완성!</h1>
+      <h2 className="py-1 font-sfpro text-2xs font-bold text-white">
+        사진과 어울리는 프레임을 선택해주세요 😀
+      </h2>
+      <div className="text-2sx mb-6 mt-4 flex h-[35px] w-[183px] flex-row items-center justify-evenly rounded-full bg-white font-sfpro font-bold">
+        {isPremiumSelected ? (
+          <>
+            <button
+              type="button"
+              onClick={() => setIsPremiumSelected(false)}
+              className="flex h-[27px] w-[86px] items-center justify-center rounded-full bg-white text-primary-middlegray"
+            >
+              기본
+            </button>
+            <div className="flex h-[27px] w-[86px] items-center justify-center rounded-full bg-primary-darkblue text-white">
+              프리미엄
+            </div>
+          </>
+        ) : (
+          <>
+            <div className="flex h-[27px] w-[86px] items-center justify-center rounded-full bg-primary-darkblue text-white">
+              기본
+            </div>
+            <button
+              type="button"
+              onClick={() => setIsPremiumSelected(true)}
+              className="flex h-[27px] w-[86px] items-center justify-center rounded-full bg-white text-primary-middlegray"
+            >
+              프리미엄
+            </button>
+          </>
+        )}
+      </div>
+
+      <div
+        className={`relative flex ${colorOfCircle === '/premiumframe2.png' ? 'h-[332px] w-[283px]' : 'h-[290px] w-[239px]'} justify-center`}
+      >
+        <Image
+          src={imageSrc}
+          alt="Sample Image"
+          width={206}
+          height={206}
+          priority
+          className="absolute mt-4"
+        />
+        {colorOfCircle && colorOfCircle !== '' && (
+          <Image
+            src={colorOfCircle}
+            alt="Selected Frame"
+            width={frameWidth}
+            height={frameHeight}
+            priority
+            className={`relative ${colorOfCircle === '/premiumframe2.png' ? '-mt-5 mb-6' : ''}`}
+          />
+        )}
+      </div>
+
+      <div
+        className={`flex flex-row gap-x-2.5 ${colorOfCircle === '/premiumframe2.png' ? '-mt-5 pb-6' : 'py-6'}`}
+      >
+        {Object.entries(frametype).map(([key, frame]) => (
+          <SelectFrame
+            key={key}
+            {...frame}
+            onClick={() => setColorOfCircle(frame.frame)}
+          />
+        ))}
+      </div>
+
+      <button
+        type="button"
+        onClick={onCaptureClick}
+        className="flex h-12 w-full items-center justify-center gap-x-3 rounded-full bg-primary-darkblue text-xl text-white"
+      >
+        <SVGDownload />
+        다운로드
+      </button>
     </div>
   );
 }
+
+export default FrameSelectView;
```

Comment:
이 부분은 그냥 정의부분과 합쳐도 좋을것 같습니다
```suggestion
export default function FrameSelectView () {}
```

Reply:
수정해두었습니다